examples/config.yml

```diff
@@ -91,6 +91,19 @@ lighthouse:
     #- "1.1.1.1:4242"
     #- "1.2.3.4:0" # port will be replaced with the real listening port
 
+  # EXPERIMENTAL: This option may change or disappear in the future.
```

Comment:
Can we document this with an "experimental" warning under https://nebula.defined.net/docs/config/?